core/src/main/resources/schema/app/20200408152731_CB-6380_Do_not_persist_telemetry_for_stack_template.sql

```diff
@@ -0,0 +1,7 @@
+-- // CB-6380 Do not persist telemetry component for stack with template type
+-- Migration SQL that makes the change goes here.
+
+DELETE FROM component WHERE componenttype = 'TELEMETRY' AND stack_id IN (SELECT id FROM stack WHERE type = 'TEMPLATE')
```

Comment:
Should this be done only for stacks that are already deleted?

Reply:
for deleted stacks, those are not template ones, and telemetry components are deleted correctly there. for template types it's not needed having connected telemetry, as if it becomes a real stack, telemetry component will be populated (based on environment + sdx response). - basically that's what i have tested, like if im not having this telemetry component, for a UI based install, the right telemetry values are populated (without having anything in the stack templates)